hmi_sdk/HMI_SDK/AppData/AppList.cpp

```diff
@@ -337,27 +335,30 @@ void AppList::getAppList(std::vector<int>& vAppIDs, std::vector<std::string>& vA
     }
 }
 
-void AppList::appUnregistered(Json::Value jsonObj)
+void AppList::appUnregistered(int appId)
 {
-    int appID = jsonObj["params"]["appID"].asInt();
-
     std::vector <AppData *>::iterator i;
```

Comment:
Should be ```const_iterator```? 
Don't use i as variable, it's not clearly express what the variable meaning. Will be better if named as `appData_it` here.